File: src/main/resources/output/IFC4X3_java/Entities/IfcRepresentationItem/IfcGeometricRepresentationItem/IfcTextLiteral/IfcTextLiteralWithExtent/IfcTextLiteralWithExtent.java
public class IfcTextLiteralWithExtent extends IfcTextLiteral {
    public IfcPlanarExtent Extent;
    public IfcBoxAlignment BoxAlignment;

    // === WHERE CLAUSES ===
    // WR31 : NOT('IFC4X3_DEV_73740fe4.IFCPLANARBOX' IN TYPEOF(Extent))
}

File: src/main/resources/output/IFC4X3_python/Entities/IfcRepresentationItem/IfcGeometricRepresentationItem/IfcTextLiteral/IfcTextLiteralWithExtent/IfcTextLiteralWithExtent.py
class IfcTextLiteralWithExtent(IfcTextLiteral):
    def __init__(self):
        self.Extent: IfcPlanarExtent = None
        self.BoxAlignment: IfcBoxAlignment = None

    # === WHERE CLAUSES ===
    # WR31 : NOT('IFC4X3_DEV_73740fe4.IFCPLANARBOX' IN TYPEOF(Extent))
